src/export/sealed.rs

```diff
@@ -0,0 +1,82 @@
+#[allow(unused_imports)]
```

Comment:
Why is this file called `sealed.rs`?

Reply:
Because the module was called like this before and I just kept the name: https://github.com/knurling-rs/defmt/blob/e021e7d013b9202f70bc762f6909401f85142c89/src/export.rs#L101

What other name do you recommend?

Reply:
Ah, that's weird. Perhaps `traits` would be better then?